dev/source/docs/setting-up-sitl-on-windows.rst

```diff
@@ -1,7 +1,7 @@
 .. _setting-up-sitl-on-windows:
 
 =============================================
-SITL on Windows in a VMWare VM (Manual Setup)
+SITL on Windows in a Virtual Machine - VM (Manual Setup)
```

Comment:
needs underlining fixed

Reply:
I fixed

Reply:
thanks you Henry, this got lost in PR stack ...